### ISSUE TITLE
`createPostResponse` fails with VersionedTransaction and address lookups not resolved

### ISSUE THREAD

**Issue by omersuve** [solana-developers/solana-actions#35]
I'm encountering an issue when trying to use the `createPostResponse` function from the Solana Actions package with a `VersionedTransaction`. (Jupiter Swap Operation in this case)

```Error: Failed to get account keys because address table lookups were not resolved```

```javascript
const { createPostResponse } = require('@solana/actions');

const connection = new Connection("https://api.mainnet-beta.solana.com");

const response = await axios.post(`${JUPITER_BASE_URL}/v6/swap`, {
  quoteResponse,
  userPublicKey,
  wrapAndUnwrapSol: true
});

const { swapTransaction } = response.data;

const transactionBuffer = Buffer.from(swapTransaction, "base64");
const versionedTransaction = VersionedTransaction.deserialize(transactionBuffer);

versionedTransaction.feePayer = new PublicKey(account);
const { blockhash } = await connection.getLatestBlockhash();
versionedTransaction.recentBlockhash = blockhash;

const responsePayload = await createPostResponse({
  fields: {
    transaction: versionedTransaction,
    message: 'You just applied swap with Jupiter!',
  },
});
```

**Log:**

```plaintext
versionedTransaction.message.addressTableLookups: [
  {
    accountKey: PublicKey [PublicKey(4i9Tt3kWnMDv1oLDyCzbrNsDY3Qh3wJRbKTwxr7913Rb)] {
      _bn: <BN: 371d71901a99b6894382204fea7560f67feff8bf914afd125ba2a6e631f38fda>
    },
    writableIndexes: [ 79, 76, 74, 77 ],
    readonlyIndexes: [ 73, 80, 82, 75, 83, 78 ]
  }
]
```

**Comment by ngundotra** [comment]
@nickfrosty ^ yea seeing this too

Can we make inserting the memo keys optional? Successfully decompiling the transaction sometimes requires having connection to resolve lookup table accounts.

```typescript
async function prepareVersionedTransaction({
  fields,
  signers,
  reference,
  actionIdentity,
}: CreateActionPostResponseArgs<VersionedTransaction>): Promise<ActionPostResponse> {
  let message = TransactionMessage.decompile(fields.transaction.message);
```

**Comment by nickfrosty** [comment]
Thanks for pointing this out! I just pushed most of a fix in v1.6.6!

It will work for all version transactions that DO NOT use the server identity. So it should work out of the box for like 99% of people (since basically no one uses server identity)

I will fix the server identity version in a follow-up fix
